other/materials_designer/create_cluster_ase.ipynb

```diff
@@ -0,0 +1,248 @@
+{
```

Comment:
<p>We don't have to put ASE in title, in the text is sufficient</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
But the filename will still have it?